packages/settings/src/reader/ReadSettings.ts

```diff
@@ -53,6 +55,11 @@ export class ReadSettings<Path = SettingPath> {
     return settings;
   }
 
+  async getPatientPortalSettings() {
+    const allSettings = await this.getAll();
+    return pick(allSettings, KEYS_EXPOSED_TO_PATIENT_PORTAL);
```

Comment:
I didn't bother setting up another cache for the portal settings when the allSettings is already cached.